pr_testing/run-pr-profiling.sh

```diff
@@ -32,6 +32,7 @@ for PROFILING_WORKFLOW in $WORKFLOWS;do
   PROF_RES="OK"
   export PROFILING_WORKFLOW
   $WORKSPACE/profiling/Gen_tool/Gen.sh $CMSSW_VERSION || true
+  get_jenkins_artifacts profiling/${CMSSW_VERSION}/${SCRAM_ARCH}/${PROFILING_WORKFLOW}/step2.root $WORKSPACE/$CMSSW_VERSION/$PROFILING_WORKFLOW/step2.root || true
```

Comment:
@gartung , so what dowe do with this step2.root file? Do we need any update in https://github.com/cms-cmpwg/profiling/blob/master/Gen_tool/runall.sh  too ?

Reply:
I added a check for the variable RUNALLSTEPS to runall.sh so it will skip the first two steps. The script will start with step3 and read in step2.root copied from the IB profiling job.

Reply:
https://github.com/cms-cmpwg/profiling/commit/a39c6cdb72abf0a71bee1a9891793b37e6a7b695